asctb-api/src/functions/api.functions.ts

```diff
@@ -1,9 +1,9 @@
-import { arrayNameMap, createObject, DELIMETER, HEADER_FIRST_COLUMN, metadataNameMap, objectFieldMap, Row } from '../models/api.model';
+import { arrayNameMap, createObject, DELIMETER, HEADER_FIRST_COLUMN, metadataArrayFields, metadataNameMap, objectFieldMap, Row, TITLE_ROW_INDEX } from '../models/api.model';
 import { fixOntologyId } from './lookup.functions';
 
 export interface ASCTBData {
   data: Row[];
-  metadata: Record<string, string>;
+  metadata: Record<string, string | string[]>;
```

Comment:
Need to update the openapi spec with this new typing -- string | string[]. I think you can use anyOf (or put the specific fields it can return and their type, which is probably better from a documentation/usability perspective).

Reply:
Added JSON schema for metadata object